unix-domain-socket/src/main/scala/akka/stream/alpakka/unixdomainsocket/scaladsl/UnixDomainSocket.scala

```diff
@@ -170,6 +170,7 @@ object UnixDomainSocket extends ExtensionId[UnixDomainSocket] with ExtensionIdPr
                       sendReceiveContext.receive = PendingReceiveAck(queue, buffer, pendingResult)
                       key.interestOps(key.interestOps() & ~SelectionKey.OP_READ)
                     } else {
+                      key.interestOps(key.interestOps() & ~SelectionKey.OP_READ)
```

Comment:
This can now move outside the conditional block given that both conditions remove the subscription.

Reply:
Yup, moving it outside the conditional would be good.

Reply:
Moved!